save-cloud-charts/save-cloud/values.yaml

```diff
@@ -202,3 +202,10 @@ demo_cpg:
   containerPort: 5500
   managementPort: 5501
   clusterIP: null
+demo:
+  name: demo
+  profile: dev
```

Comment:
why do we enable `dev` profile on k8s?

Reply:
Just a default value, for Minikube for example. It's overridden for actual prod

Reply:
oh.. it's overridden by `values-cce.yaml`. Got it